src/manager/manager.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 from classes.vehicle import Vehicle
 from .command import Command
+from ..classes.route import route_position_to_world_position
```

Comment:
This was giving issues. Changing to:
`from classes.route import...` 
fixed the issue.